content/data-types/exercises/_index.md

```diff
@@ -0,0 +1,170 @@
+   
+   {{% expand "Click Here for Answer" %}}
+
+   ```csharp
+      Console.WriteLine("The area of the rectangle is: " + area);
+   ```
+
+   {{% /expand %}}
+
+   f. Run the program to verify your code.
+
+## More on Numeric Types
+
+1. Write a program that asks a user for the number of
+   miles they have driven and the amount of gas they’ve consumed (in
+   gallons), and print their miles-per-gallon.
+
+{{% expand "Click Here for Answer" %}}
+
+   ```csharp {linenos=table}
+      Console.WriteLine("How many miles did you drive on your trip?");
+      string mi = Console.ReadLine();
+      int miles = Int32.Parse(mi);
+
+      Console.WriteLine("How many gallons of gas did you use?");
+      string gal = Console.ReadLine();
+      int gallons = Int32.Parse(gal);
+
+      int mpg = miles / gallons;
+      Console.WriteLine("The MPG for the trip was: " + mpg);
+   ```
+
+{{% /expand %}}
+
+## Strings
+
+1. The first sentence of *Alice’s Adventures in Wonderland*
+   is below. Store this sentence in a string, and then prompt the user
+   for a term to search for within this string. Print whether or not the
+   search term was found. Make the search case-insensitive, so that searching
+   for "alice", for example, prints `true`.
+      ```
+      Alice was beginning to get very tired of sitting by her sister on the
+      bank, and of having nothing to do: once or twice she had peeped into the
+      book her sister was reading, but it had no pictures or conversations in
+      it, 'and what is the use of a book,' thought Alice 'without pictures or
+      conversation?'
+      ```
+
+   {{% expand "Click Here for Answer" %}}
+
+   ```csharp {linenos=table} 
+      string alice = @"Alice was beginning to get very tired of sitting by her sister on the
+      bank, and of having nothing to do: once or twice she had peeped into the
+      book her sister was reading, but it had no pictures or conversations in
+      it, 'and what is the use of a book,' thought Alice 'without pictures or
+      conversation?'";
+      
+      Console.WriteLine(alice);
+      Console.WriteLine("What sentence would you like to look for in the sentence above?");
+      string searchTerm = Console.ReadLine();
+      string compSearchTerm = searchTerm.ToLower();
+      string compAlice = alice.ToLower();
+
+      if (compAlice.IndexOf(compSearchTerm, 0) != -1)
+      {
+         Console.WriteLine("true");
+      }
+      else 
+      {
+         Console.WriteLine("false");
+      }
+   ```
+
+   {{% /expand %}}
+
+1. Extend the previous exercise. Assume the user enters a word that is in the sentence. Print out its index within the string and its length. Next, remove the word from the string and print the sentence again to confirm your code. Remember that strings are *immutable*, so you will need to reassign the old sentence variable or create a new one to store the updated phrase.
```

Comment:
Noticed that if you use Alice it only removes the first instance.  I'm assuming they do not have the knowledge yet to correct for this?  If they do could this be reworked? Or if not, add a note?

Reply:
That is expected and we do not need to add a note for students as they have learned about the logic of removing one instance and multiple instances of substrings in Unit 1.